packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -1,45 +1,136 @@
 <template>
-    <div
-        :class="$style['o-link']"
-        data-test-id="link"
-    >
-        {{ copy.text }}
-    </div>
+    <span>
+        <a
+            :class="[
+                $style['o-link'], {
+                    [$style['o-link--bold']]: isBold,
+                    [$style['o-link--noDecoration']]: !hasTextDecoration,
+                    [$style['o-link--full']]: isFullWidth,
+                    [$style['o-link--noBreak']]: noLineBreak
+                }]"
+            data-test-id="link-component"
+            aria-describedby="description"
+            :target="target"
+            :rel="rel"
+            v-bind="$attrs"
+        >
+            <slot />
+        </a>
+        <span
+            v-if="ariaDescription"
+            id=description
```

Comment:
```suggestion
            id="description"
```

Reply:
🤦

Reply:
Actually on this - an `id` attribute must be unique. So assuming there may be more than one link per page, this should probably be a system-generated identifier (maybe a slugified version of the link text?) rather than a fixed string.

Reply:
Updated this now, Thanks :)